server.js

```diff
@@ -120,7 +120,7 @@ module.exports = function(options) {
           schema: {
             id: Joi.string(),
             frequency: Joi.string().valid("monthly", "one-time"),
-            currency: Joi.string().min(3).max(3),
+            currency: Joi.any().valid(currencyFor.stripe).required(),
```

Comment:
I think at this point, this is just covering ourselves to not break what we pass back.